frontend/index.html

```diff
@@ -0,0 +1,34 @@
+<script>
+  async function callApi(path) {
+    const url = "http://localhost" + path;
+    try {
+      const response = await fetch(url, {
+        headers: {
+          'accept': 'application/json',
+          'X-API-Key': 'abc123'
```

Comment:
bit of a longshot, but is there a simple way to read this directly in from the `.env` file? That way we don't need to worry about keeping them in sync for testing.

Reply:
But I could make an HTML input field where the user could input the key and then run the call with it to load the data.

Reply:
No that's ok, let's just handle that when we make this into a more legitimate front-end. 
This is great for now for testing.